chasm/lib/activity/activity.go

```diff
@@ -56,23 +63,31 @@ func NewStandaloneActivity(
 		return nil, err
 	}
 
-	return &Activity{
+	activity := &Activity{
 		ActivityState: &activitypb.ActivityState{
 			ActivityType:    request.ActivityType,
 			ActivityOptions: request.Options,
 			Priority:        request.Priority,
 		},
 		Attempt: chasm.NewDataField(ctx, &activitypb.ActivityAttemptState{
-			Count:           1,
-			LastStartedTime: timestamppb.Now(),
+			Count: 1,
 		}),
 		RequestData: chasm.NewDataField(ctx, &activitypb.ActivityRequestData{
 			Input:        request.Input,
 			Header:       request.Header,
 			UserMetadata: request.UserMetadata,
 		}),
 		Visibility: chasm.NewComponentField(ctx, visibility),
-	}, nil
+	}
+
+	ctx.AddTask(
+		activity,
+		chasm.TaskAttributes{},
+		&activitypb.ActivityStartExecuteTask{
+			Attempt: 1,
```

Comment:
This should always be taken from the attempt data field even if we know it's 1 here just for completeness IMHO.